views/groups.handlebars

```diff
@@ -1,20 +1,24 @@
-<body>
-  <nav class="navbar navbar-default">
-    <div class="container-fluid">
-      <div class="navbar-header">
+<body><div id="groups-whole-page">
+  <div class="jumbotron">
+  <nav class="navbar navbar-default offset-1">
+      <div class="navbar-header">navbar stuff soon to be added here
       </div>
-    </div>
   </nav>
+   <h1 class="offset-1">CREATE A NEW LEAGUE!</h1>
+  </div>
+
+{{!-- for now I'm borrowing the same ID tag used on the members page to get a similar look. -sam --}}
+<div class="col-10 offset-1" id="body-box-members">
   <div class="container">
-      <h1>CREATE A NEW LEAGUE!</h1>
+     
     <div class="row">
       <div class="col-md-6 col-md-offset-3">
-          <div>
-            <label for="InputGroupName">Enter the name of your new league.</label>
-            <input type="test" id="group-input" placeholder="Group Name"></input>
-          </div>
+          {{!-- <div> --}}
```

Comment:
Plenty more to do in the future as I try to get divs spaced in a better manor, but for now it helped to comment out this <div>